src/hooks/pages/useHeader.ts

```diff
@@ -32,13 +33,12 @@ export type UseHeaderReturn = UseRoutesReturn & {
 }
 
 export function useHeader(): UseHeaderReturn {
-  const [state] = useAppState()
-  const {
-    provider,
-    blockchain,
-    account,
-    balance: accountBalance,
-  } = state.connection
+  const [
+    {
+      connection: { provider, blockchain, account, balance: accountBalance },
```

Comment:
Looks like this will cause linter errors. Make sure you run `lint:fix` before pushing